packages/requestNetworkSmartContracts/migrations/2_deploy_contracts.js

```diff
@@ -27,11 +28,11 @@ function createInstances() {
   });
 }
 
-function setupContracts({ ethereum: ethereumInstance, core: coreInstance }) {
+function setupContracts({ ethereum: ethereumInstance, core: coreInstance, erc20: erc20Instance }) {
   return Promise.all([
     coreInstance.adminAddTrustedCurrencyContract(ethereumInstance.address),
     ethereumInstance.setFeesPerTenThousand(feesPerTenThousand),
-    ethereumInstance.setMaxCollectable(maxFees)
+    ethereumInstance.setMaxCollectable(maxFees),
```

Comment:
There's an issue here:
The param `erc20Instance` is given line 18. It comes from the return of `createInstances`. `createInstances` doesn't return the erc20 contract